### DISCUSSION TITLE
2Checkout Connector

### DISCUSSION THREAD

**Discussion by ammo8600** [juspay/hyperswitch#4871] [Ideas / Feature Requests]
2Checkout.com connector is what's holding us back. Would love to see it.

**Comment by gorakhnathy7** [comment]
Hey @ammo8600, thanks for this feature request.

You correctly mentioned, We currently do not have 2Checkout with us. However, this can be discussed internally and added to our Roadmap.

Can you elaborate further on the use-case for which 2Checkout is required?

**Reply by ammo8600** [reply]
Basically 2Checkout serves as a proxy to those merchants in countries where Paypal is not available.
